### PR TITLE
Add support for per-sequence animation defaults

### PR DESCRIPTION
**Pull request by pchote**
This adds support for a "Defaults" section in each sequence, and removes redundancies from the TS sequences in preparation for tileset-specific artwork.

### CODE REVIEW

File: mods/ts/sequences/misc.yaml [hunk 1]
```diff
@@ -1,44 +1,24 @@
...
 		Offset: 0, -12
+	build-valid-interior: place
 	build-valid-snow: place
-		Start: 0
-		Offset: 0, -12
 	build-valid-temperat: place
-		Start: 0
-		Offset: 0, -12
 	build-invalid: place
-		Start: 1
```
Comment (abcdefg30) [inline mods/ts/sequences/misc.yaml]: Is this change from `Start: 1` to `0` intentional?

Comment (pchote) [inline mods/ts/sequences/misc.yaml]: It wasn't.  Good catch, and fixed.

**Comment by abcdefg30** [comment]
Yaml looks ok and didn't find regressions ingame. :+1: / :white_check_mark:

File: mods/ts/sequences/structures.yaml [hunk 2]
```diff
@@ -1,781 +1,598 @@
...
 		Start: 1
 		ShadowStart: 4
-		Offset: 0, -12
 	dead: ntsam
 		Start: 2
 		ShadowStart: 5
-		Offset: 0, -12
 		Tick: 400
 	turret: gtctwr_d
 		Start: 0
```
Comment (Mailaender) [inline mods/ts/sequences/structures.yaml]: I guess this one is redundant, too.

**Comment by pchote** [comment]
Removed some more redundant entries.

**Comment by Mailaender** [comment]
Quite the line removal. :chart_with_downwards_trend:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/8c44a8309acbfd1de9269f2999ded0ec59913d5e...d011b4139fbd6e2640d63cfed9c679cf7a39eee8)

**Pull request merged**